spec/feature/takeaway_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'takeaway'
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+describe Takeaway do
+  subject(:takeaway) {described_class.new}
+  it 'returns a list of dishes and their respective prices' do
+    expect(takeaway).to respond_to(:menu)
+  end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+
+  describe '#place_order' do
+    it 'stores an order with the name and quantity of dishes' do
+      test_order = {:dish1=>3,:dish2=>2,:dish3=>5,:dish4=>1}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)